src/storage/storage.cc

```diff
@@ -363,7 +363,7 @@ Status Storage::CreateBackup() {
   }
 
   std::unique_ptr<rocksdb::Checkpoint> checkpoint_guard(checkpoint);
-  s = checkpoint->CreateCheckpoint(tmpdir, config_->rocks_db.write_buffer_size * MiB);
+  s = checkpoint->CreateCheckpoint(tmpdir, config_->rocks_db.write_buffer_size * MiB, sequence_number);
```

Comment:
Just a question:

```
  // sequence_number_ptr: if it is not nullptr, the value it points to will be
  // set to a sequence number guaranteed to be part of the DB, not necessarily
  // the latest. The default value of this parameter is nullptr.
```

From rocksdb, seems this might a bit less than backup? Or I misunderstanding this?

Reply:
Maybe, I didn't investigate when it would be less than the DB.

Reply:
It was introduced here: https://github.com/facebook/rocksdb/commit/3ffb3baa0b5ce782ea36c8c53adb211127a92dcc

Maybe we should notice user to pay-attention to that, replaying kv might not harm

Reply:
I have confirmed the implementation, it is the latest sequence number while starting to create the checkpoint, so it should be less the current backup DB if has any writes during this process.

Reply:
Yeah I guess maybe it's not atomic between:

```c++
  *sequence_number = db_->GetLatestSequenceNumber();

  LiveFilesStorageInfoOptions opts;
  opts.include_checksum_info = get_live_table_checksum;
  opts.wal_size_for_flush = log_size_for_flush;

  std::vector<LiveFileStorageInfo> infos;
  {
    Status s = db_->GetLiveFilesStorageInfo(opts, &infos);
```